modules/hcp-dc-tls.adoc

```diff
@@ -0,0 +1,47 @@
+// Module included in the following assemblies:
+//
+// * hosted_control_planes/hcp-disconnected/hcp-deploy-dc-bm.adoc
+// * hosted_control_planes/hcp-disconnected/hcp-deploy-dc-virt.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="hcp-dc-tls_{context}"]
+= Configuring TLS certificates for a disconnected installation of {hcp}
+
+Several TLS certificates are involved in the process to configure {hcp} in a disconnected environment. To add a Certificate Authority (CA) to the management cluster, you need to modify the content of the following files in the {product-title} control plane and worker nodes:
+
+* `/etc/pki/ca-trust/extracted/pem/`
+* `/etc/pki/ca-trust/source/anchors`
+* `/etc/pki/tls/certs/`
+
+.Procedure
+
+To add a CA to the management cluster, complete the following steps:
+
+. Complete the steps in _Updating the CA bundle_. That method involves using the `image-registry-operator`, which deploys the CAs to the {product-title} nodes.
+
+. If that method does not apply to your situation, check whether the `openshift-config` namespace in the management cluster contains a config map named `user-ca-bundle`.
++
+* If the namespace contains that config map, enter the following command:
++
+[source,terminal]
+----
+## REGISTRY_CERT_PATH=<PATH/TO/YOUR/CERTIFICATE/FILE>
+export REGISTRY_CERT_PATH=/opt/registry/certs/domain.crt
+
+$ oc create configmap user-ca-bundle -n openshift-config --from-file=ca-bundle.crt=${REGISTRY_CERT_PATH}
+----
+
+* If the namespace does not contain that config map, enter the following command:
++
```

Comment:
Note to technical reviewers: We need to break up the following code block per OCP doc standards. We need one command per code block.